### DISCUSSION TITLE
FAILED: Could not determine revision id from filename env.py when the migrations directory is different from alembic

### DISCUSSION THREAD

**Discussion by sidux** [sqlalchemy/alembic#1346] [Usage Questions]
**To Reproduce**
on a new project : 
- `alembic init migrations`
- `alembic revision -m "message" --autogenerate`

I get the error `FAILED: Could not determine revision id from filename env.py`

Everything is okay if I do `alembic init alembic` instead.

**Comment by zzzeek** [comment]
hi -

there is nothing special about the name "migrations".

if you are calling `alembic init` a second time, make sure you delete / move your `alembic.ini` file that would have been there from the previous run so you get a new file correctly formatted
